samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -139,22 +142,28 @@ Using the built-in temperature sensor of the `Nordic Thingy:91`_ requires a `dev
 The devicetree overlay file is automatically applied during compilation whenever the ``thingy91_nrf9160_ns`` target is selected.
 The required Kconfig options are implicitly enabled by :ref:`CONFIG_TEMP_DATA_USE_SENSOR <CONFIG_TEMP_DATA_USE_SENSOR>`.
 
+Note that for temperature readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
+This is performed by the :file:`src/connection.c` file.
+
 .. _nrf_cloud_mqtt_multi_service_location_tracking:
 
 Location tracking
 =================
 
 All matters concerning location tracking are handled in the :file:`src/location_tracking.c` file.
 For the most part, this amounts to setting up a periodic location request, and then passing the results to a callback configured by the :file:`src/application.c` file.
-Additionally, the following two tasks which must be implemented:
+Additionally, the following two tasks are performed:
 
-* The :ref:`lib_location` library requires assisted GPS (A-GPS) data to have the fastest possible `time-to-first-fix <TTFF_>`_.
+* The :ref:`lib_location` library requires GPS assistance data (either A-GPS, P-GPS, or both) to have the fastest possible `time-to-first-fix <TTFF_>`_.
   The library requests this data automatically, but has no way to automatically receive the response.
-  Therefore, all MQTT messages received from `nRF Cloud`_ must be be passed directly through to the :ref:`lib_nrf_cloud_agps` library to be checked for A-GPS data.
-  The :c:func:`location_agps_data_handler` function handles this task, and MQTT messages are sent directly to it from the :file:`src/connection.c` file.
+  Therefore, all MQTT messages received from `nRF Cloud`_ must be be passed directly through to the :ref:`lib_nrf_cloud_agps` and :ref:`lib_nrf_cloud_pgps` libraries to be checked for A-GPS and P-GPS data, respectively.
+  The :c:func:`location_assistance_data_handler` function handles this task, and MQTT messages are sent directly to it from the :file:`src/connection.c` file.
 * The GNSS module cannot obtain a fix without the antenna first being properly configured.
   This configuration is performed by the :c:func:`gnss_antenna_configure` function.
 
+Note that for location readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
+This is performed by the :file:`src/connection.c` file.
```

Comment:
```suggestion
.. note::
   For location readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
   This is performed by the :file:`src/connection.c` file.
```

Reply:
I hesitate to put this in a proper note box, since there is no direct action required by the user under any circumstance. It is more meant as a theoretical footnote. Or would that still qualify for inclusion in a note box?